### ISSUE TITLE
Error copying vagrant box

### ISSUE THREAD

**Issue by qrevel** [bacongravy/macinbox#15]
By default, boxes are copied to `~/.vagrant.d/boxes`.

So when `~/.vagrant.d/boxes` doesn't exist, we get the following error:

``` 
• Installing box...
  + Copying box to ~/.vagrant.d/boxes...
    - Error: ~/.vagrant.d/boxes not found
```

I see three solutions here:

- Create the folder if not found
- Use the [VAGRANT_HOME](https://www.vagrantup.com/docs/other/environmental-variables.html#vagrant_home) env var
- Use `vagrant box add`

**Comment by bacongravy** [comment]
Hello @qrevel, and thank you for this report.  I chose not to use `vagrant box add` because it makes the whole process take longer and use more disk space, as the box must be tar'd up and then untar'd again.

I like the suggestion to use the `VAGRANT_HOME` env var when available, falling back on `~/.vagrant.d/boxes` as the default.  I'm reluctant to create the folder if it is not found.  I'd prefer to check for the existence of this folder when `macinbox` starts up, before any disk images are made, and to abort early with an error if it doesn't exist.

**Comment by qrevel** [comment]
Ok, currently I'm just creating the `~/.vagrant.d/boxes` folder as a workaround and move the box in my `VAGRANT_HOME` after.

**Comment by bacongravy** [comment]
I've addressed this with https://github.com/bacongravy/macinbox/commit/97e3a0e7967add221600463373cd89021107aee3. The VAGRANT_HOME environment variable is used if it is set, and the `boxes` directory is created inside of it, if necessary.

Note that you do need to make extra effort to get the environment variable across the `sudo` barrier, e.g.:

    $ sudo "VAGRANT_HOME=${VAGRANT_HOME}" macinbox

I haven't released 3.0 yet but you can build and install the gem yourself if you follow the instructions in the README.

**Comment by qrevel** [comment]
Ok thanks I'll try it 👍
